storage/src/main/java/org/apache/uniffle/storage/common/LocalStorage.java

```diff
@@ -270,6 +269,14 @@ public void updateDiskFree(long free) {
     this.diskFree = free;
   }
 
+  public void updateServiceUsed(long used) {
```

Comment:
updateServiceUsed -> updateServiceUsedSize

It is better? 

BTW. What does `service` here mean? Do we have a better name?

Reply:
> BTW. What does `service` here mean? Do we have a better name?

This variable means the total size of files written by this shuffle server instance. Do you have any good ideas?